include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -261,9 +261,11 @@ ORT_API(OrtRunOptions*, OrtCreateRunOptions);
 
 ORT_API_STATUS(OrtRunOptionsSetRunLogVerbosityLevel, _In_ OrtRunOptions*, unsigned int);
 ORT_API_STATUS(OrtRunOptionsSetRunTag, _In_ OrtRunOptions*, _In_ const char* run_tag);
+ORT_API(void, OrtRunOptionsSetCacheFeedsFetchesInfoEnabled, _In_ OrtRunOptions* options, bool value);
```

Comment:
OrtRunOptionsSetTerminate uses an 'int' to set a boolean flag. We should be consistent with the usage of boolean types. Are we using int or bool?

Reply:
'int' is probably cleaner.